.github/workflows/format-check.yml

```diff
@@ -1,4 +1,4 @@
-name: Check Format
+name: Check/apply clang-format
```

Comment:
```suggestion
name: apply clang-format
```

Reply:
Ah ok, I just wanted to have the case when the bot will not make a commit, but ok I'll remove the / now